manifest/Manifest.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<title>Test that "manifest" is a supported value for the `rel` of a `link`</title>
+<link rel="help" href="https://w3c.github.io/manifest">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+test(() => {
+    document.createElement("link").relList.supports("manifest")
+}, 'link element should support a rel value of "manifest".');
```

Comment:
Try to avoid RFC2119 keywords in these assertions, as we are generally testing "MUST" things.

```suggestion
}, 'link element supports a rel value of "manifest".');
```